src/main/resources/hudson/security/table.js

```diff
@@ -197,64 +199,43 @@ Behaviour.specify(".global-matrix-authorization-strategy-table TD.stop A.migrate
 
     return false;
   };
-  e = null; // avoid memory leak
 });
 
-/*
- * Determine which attribute to set tooltips in. Changed in Jenkins 2.379 with Tippy and data-html-tooltip support.
- */
-function getTooltipAttributeName() {
-  let coreVersion = document.body.getAttribute('data-version');
-  if (coreVersion === null) {
-    return 'tooltip'
-  }
-  // TODO remove after minimum version is 2.379 or higher
-  let tippySupported = coreVersion >= '2.379';
-  return tippySupported ? 'data-html-tooltip' : 'tooltip';
-}
-
 /*
  * Whenever permission assignments change, this ensures that implied permissions get their checkboxes disabled.
  */
 Behaviour.specify(".global-matrix-authorization-strategy-table td input", 'GlobalMatrixAuthorizationStrategy', 0, function(e) {
-  var table = findAncestor(e, "TABLE");
+  const table = findAncestor(e, "TABLE");
   if (table.classList.contains('read-only')) {
     // if this is a read-only UI (ExtendedRead / SystemRead), do not enable checkboxes
     return;
   }
 
-  var tooltipAttributeName = getTooltipAttributeName();
+  const tooltipAttributeName = 'data-html-tooltip';
 
-  var impliedByString = findAncestor(e, "TD").getAttribute('data-implied-by-list');
-  var impliedByList = impliedByString.split(" ");
-  var tr = findAncestor(e,"TR");
+  const impliedByString = findAncestor(e, "TD").getAttribute('data-implied-by-list');
+  const impliedByList = impliedByString.split(" ");
+  const tr = findAncestor(e, "TR");
   e.disabled = false;
-  let tooltip = YAHOO.lang.escapeHTML(findAncestor(e, "TD").getAttribute('data-tooltip-enabled'));
-  e.setAttribute(tooltipAttributeName, tooltip); // before 2.335 -- TODO remove once baseline is new enough
```

Comment:
Note for later: This line and the corresponding removed line 243 break the Add buttons. Rest of the code needs to be adapted.

Reply:
https://github.com/jenkinsci/matrix-auth-plugin/pull/152/commits/36597c3612ab5780d413c7c319cc707453d46b35